packages/cli/src/UserManagement/Interfaces.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
ref previous comments about file wide rule disablings

https://github.com/n8n-io/n8n/pull/2636/files#r798351153

Reply:
Removed in `user-management-backend-testing-owner-setup`